### DISCUSSION TITLE
Bundle a different emoji font with a custom theme

### DISCUSSION THREAD

**Discussion by montovaneli** [fyne-io/fyne#4744] [Q&A] [answered]
How can I bundle a different emoji font with a custom theme? I tried exploring the creation of the defaultTheme to understand how it works, but at no time does the builtinTheme present a reference for the emoji font. Could you please clarify if it is possible and, if so, how to do it?

Thank you.

**Comment by andydotxyz** [comment, marked as answer]
It is not currently possible - the emoji and symbol fonts we fall back to are coded in.
One issue we have about opening this up is that emoji fonts have vastly varying content and some of the complexity involved is beyond the SVG parser we use at this time - so you might find it doesn't work anyway.
